docs/library/platform.rst

```diff
@@ -0,0 +1,25 @@
+:mod:`platform` -- access to underlying platform’s identifying data
+===================================================================
+
+.. module:: platform
+   :synopsis: access to underlying platform’s identifying data
+
+|see_cpython_module| :mod:`python:platform`.
+
+This module tries to retrieve as much platform-identifying data as
+    possible. It makes this information available via function APIs.
+
+Functions
+---------
+
+.. function:: platform()
+
+   Returns a single string identifying the underlying platform.
+
+.. function:: python_compiler()
+
+   Returns a string identifying the compiler used for compiling MicroPython.
+
+.. function:: libc_ver()
+
+   Returns a tuple of strings (lib, version) which default to the given parameters in case the lookup fails.
```

Comment:
```suggestion
   Returns a tuple of strings (lib, version) which defaults to the given parameters in case the lookup fails.
```

Reply:
What are the "given parameters"? I doesn't look like this function takes any parameters.

Reply:
I've revised the description for `libc_ver()`, let me know if there are any issues with it.